content/applications/inventory_and_mrp/barcode/setup/hardware.rst

```diff
@@ -4,64 +4,78 @@ Set up your barcode scanner
-    It consists of a small computer with a built-in barcode scanner.
-    This one can turn out to be a very productive solution, however
-    you need to make sure that is is capable of running Odoo smoothly.
-    The most recent models using Android + Google Chrome or Windows +
-    Internet Explorer Mobile should do the job. However, due to the
-    variety of models and configurations on the market, it is
-    essential to test it first.
-
-Configure your barcode scanner
-==============================
+Follow this guide to choose and set up a barcode scanner compatible with Odoo's *Inventory* and
+*Barcode* apps.
+
+.. image:: hardware/barcode-scanner.png
+   :align: center
+   :alt: An image of an example barcode scanner.
+
+Scanner types
+=============
+
+Before setting up a barcode scanner, it is important to determine which scanner type best meets the
+needs of the business. There are three main types, each with their own benefits and use cases:
+
+- **USB scanners** are connected to a computer, and are suitable for businesses that scan products
+  at a fixed location, like at the checkout in a grocery store. Ensure the chosen USB scanner is
+  compatible with the keyboard layout of the computer.
+
+- **Bluetooth scanners** pair with a smartphone or tablet, making them an ideal cost-effective and
+  portable barcode scanner option. In this scenario, Odoo is installed on the smartphone, allowing
+  warehouse operators to handle operations, and check stock directly through their mobile devices.
+
+- **Mobile computer scanners** are mobile devices with a built-in barcode scanner. First, ensure the
+  device can run the Odoo mobile app properly. Recent models that use Android OS with the Google
+  Chrome browser, or Windows OS with Microsoft Edge, should work. However, testing is crucial due to
+  the variety of available models and configurations.
+
+Configuration
+=============
+
+When setting up the barcode scanner, ensure the following configurations are correctly set to ensure
+the scanner can interpret barcodes properly in Odoo.
 
 Keyboard layout
 ---------------
 
-.. image:: hardware/hardware01.png
-    :align: center
+When using a USB barcode scanner, match its keyboard layout with the operating system's layout for
+proper interpretation of characters. Generally, the scanning mode should be set to accept a USB
+keyboard (HID), with the language set based on the keyboard that is in use.
 
-An USB barcode scanner needs to be configured to use the same keyboard
-layout as your operating system. Otherwise, your scanner won't translate
-characters correctly (replacing a 'A' with a 'Q' for example). Most
-scanners are configured by scanning the appropriate barcode in the user
-manual.
+When configuring the keyboard layout for a Zebra scanner, scan the keyboard wedge barcode in the
+user manual for the barcode scanner.
+
+.. image:: hardware/keyboard-barcode.png
```

Comment:
are these actually scannable or useful for configuration?
if so that should be more explicit...could do so in the instructions before the image or do another figure directive with caption here.